packages/reports/addon/components/grouped-list.ts

```diff
@@ -49,12 +50,27 @@ export default class GroupedListComponent extends Component<Args> {
     const items: Array<
       { name: string; length: number; isOpen: boolean; isGroup: boolean } | Record<string, string>
     > = [];
-    return Object.keys(groupedItems).reduce((items, name) => {
-      const groupItems = groupedItems[name];
-      const isOpen = groupConfigs[name] || shouldOpenAllGroups;
 
-      items.push({ name, length: groupItems.length, isOpen, isGroup: true });
-      if (isOpen) {
+    // make categories into sorted array
+    let sortedCategories = Object.entries(groupedItems).sort(function (a, b) {
+      if (a[0] === `Uncategorized`) {
+        return 1;
+      } else if (b[0] === `Uncategorized`) {
+        return -1;
+      } else {
+        return a[0].localeCompare(b[0]);
+      }
+    });
+
+    // flatten category headers and items into single list
+    return sortedCategories.reduce((items, ele) => {
+      const groupItems = ele[1];
+      const name = ele[0];
```

Comment:
Let's make this a forEach
```suggestion
    return sortedCategories.forEach(([name, groupItems]) => {
```